### DISCUSSION TITLE
Is it possible to validate while last field is being filled when mode is `onTouched`?

### DISCUSSION THREAD

**Discussion by AlexMcComb** [react-hook-form/react-hook-form#8810] [Q&A] [answered]
I have my mode set to `onTouched` to validate the fields on blur. I also have my Submit form button to be disabled unless the form `isValid`. The problem is that when the user is in the last field and typing AND it's valid (and all other fields are valid), the submit button is still disabled. I would like it to be enabled at that point and the field to be validated.

I've used `trigger()` within the `onChange` on each field, but `isValid` still only fires on blur of the fields.

**Comment by bluebill1049** [comment]
trigger should update isValid, could you share a simple codesandbox?

**Reply by AlexMcComb** [reply]
Solved with this solution: https://stackoverflow.com/questions/66982034/enable-disable-submit-button-with-react-hook-form-in-onblur-mode
I also used `mode: all` so I still would get validation if they didn't "touch" the field `onBlur`

**Comment by AlexMcComb** [comment, marked as answer]
https://github.com/react-hook-form/react-hook-form/discussions/8810#discussioncomment-3360984
